tests/block_on.rs

```diff
@@ -16,3 +16,62 @@ fn panic() {
         panic!("boom");
     });
 }
+
+#[cfg(feature = "unstable")]
+#[test]
+fn nested_block_on_local() {
+    let x = task::block_on(async {
+        let a =
+            task::block_on(async { task::block_on(async { async_std::future::ready(3).await }) });
+        let b = task::spawn_local(async {
+            task::block_on(async { async_std::future::ready(2).await })
+        })
+        .await;
```

Comment:
This is kind of hard to follow. Can we split this on multiple lines or make it a one-liner? Same with the other statements below.

Maybe importing `use async_std::future` would be enough: that way we can write `future::ready(1).await` which will likely be enough to make this a one-liner.

Reply:
fixed